website/docs/r/data_protection_backup_policy_postgresql_flexible_server.html.markdown

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "DataProtection"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_protection_backup_policy_postgresql_flexible_server"
+description: |-
+  Manages a Backup Policy to back up PostgreSQL Flexible Server.
+---
+
+# azurerm_data_protection_backup_policy_postgresql_flexible_server
+
+Manages a Backup Policy to back up PostgreSQL Flexible Server.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_data_protection_backup_vault" "example" {
+  name                = "example-backup-vault"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  datastore_type      = "VaultStore"
+  redundancy          = "LocallyRedundant"
+}
+
+resource "azurerm_data_protection_backup_policy_postgresql_flexible_server" "example" {
+  name                            = "example-backup-policy"
+  vault_id                        = azurerm_data_protection_backup_vault.example.id
+  backup_repeating_time_intervals = ["R/2021-05-23T02:30:00+00:00/P1W"]
+  time_zone                       = "India Standard Time"
+
+  default_retention_rule {
+    life_cycle {
+      duration        = "P4M"
+      data_store_type = "VaultStore"
+    }
+  }
+
+  retention_rule {
+    name     = "weekly"
+    duration = "P6M"
+    priority = 20
+
+    criteria {
+      absolute_criteria = "FirstOfWeek"
+    }
+  }
+
+  retention_rule {
+    name     = "thursday"
+    duration = "P1W"
+    priority = 25
+
+    criteria {
+      days_of_week           = ["Thursday"]
+      scheduled_backup_times = ["2021-05-23T02:30:00Z"]
+    }
+  }
+
+  retention_rule {
+    name     = "monthly"
+    duration = "P1D"
+    priority = 15
+
+    criteria {
+      weeks_of_month         = ["First", "Last"]
+      days_of_week           = ["Tuesday"]
+      scheduled_backup_times = ["2021-05-23T02:30:00Z"]
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Backup Policy PostgreSQL Flexible Server. Changing this forces a new resource to be created.
```

Comment:
I think this resource is a policy, not a server? If so, please also update other places.
```suggestion
* `name` - (Required) Specifies the name of the Backup Policy for the PostgreSQL Flexible Server. Changing this forces a new resource to be created.
```

Reply:
Updated